tee-worker/ts-tests/examples/direct-invocation/util.ts

```diff
@@ -93,6 +93,26 @@ export const createSignedTrustedCall = (
     });
 };
 
+export const createSignedTrustedGetter = (
+    parachain_api: ApiPromise,
+    trustedGetter: [string, string],
+    account: KeyringPair,
+    params: Array<any>
```

Comment:
`params: Array<any> | any` ?

then you woudn't need to check `params.length` and extract first element in case of `len == 1`.

Reply:
`params: Array<any> | any` would be the same as `params: any` but more noisy :P

Reply:
next step is to get rid of `any` :)